schemas/MobileScreen.json

```diff
@@ -64,7 +64,8 @@
         {"const": "Invites", "description": "Room accessed via space bottom sheet list."},
         {"const": "CreateSpace", "description": "The screen shown to create a new space."},
         {"const": "LocationSend", "description": "The screen shown to share location."},
-        {"const": "LocationView", "description": "The screen shown to view a shared location."}
+        {"const": "LocationView", "description": "The screen shown to view a shared location."},
+        {"const": "CreatePollView", "description": "The screen shown to create or edit a poll."}
```

Comment:
Just a remark: don't we want to do the diff between creating and editing?

Reply:
Yes thanks, initially only poll creation will make the cut so a `EditPollView` will be added later.